bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/types/BTypes.java

```diff
@@ -41,7 +41,7 @@
     public static BType typeXMLAttributes = new BXMLAttributesType(TypeConstants.XML_ATTRIBUTES_TNAME, null);
     // public static BType typeIterator = new BIteratorType(TypeConstants.ITERATOR_TNAME, null);
     // public static BType typeChannel = new BChannelType(TypeConstants.CHANNEL, null);
-    // public static BErrorType typeError = new BErrorType(TypeConstants.ERROR, typeString, typeMap, null);
+    public static BErrorType typeError = new BErrorType(TypeConstants.ERROR, null, typeString, typeMap);
```

Comment:
If we do not need the package path, shall we remove it from the constructor itself.

Reply:
Fixed